jwql/tests/test_bad_pixel_monitor.py

```diff
@@ -23,20 +23,18 @@
 
 from jwst.datamodels import dqflags
 
-from jwql.database.database_interface import NIRCamBadPixelQueryHistory, NIRCamBadPixelStats
-from jwql.database.database_interface import NIRISSBadPixelQueryHistory, NIRISSBadPixelStats
-from jwql.database.database_interface import MIRIBadPixelQueryHistory, MIRIBadPixelStats
-from jwql.database.database_interface import NIRSpecBadPixelQueryHistory, NIRSpecBadPixelStats
-from jwql.database.database_interface import FGSBadPixelQueryHistory, FGSBadPixelStats
-
-
-ON_JENKINS = '/home/jenkins' in os.path.expanduser('~')
-
-if not ON_JENKINS:
+ON_GITHUB_ACTIONS = '/home/runner' in os.path.expanduser('~')
+
+if not ON_GITHUB_ACTIONS:
+    from jwql.database.database_interface import NIRCamBadPixelQueryHistory, NIRCamBadPixelStats
+    from jwql.database.database_interface import NIRISSBadPixelQueryHistory, NIRISSBadPixelStats
+    from jwql.database.database_interface import MIRIBadPixelQueryHistory, MIRIBadPixelStats
+    from jwql.database.database_interface import NIRSpecBadPixelQueryHistory, NIRSpecBadPixelStats
+    from jwql.database.database_interface import FGSBadPixelQueryHistory, FGSBadPixelStats
```

Comment:
Is there a reason why these imports need to be under the `if not ON_GITHUB_ACTIONS` conditional?

Reply:
Hey @bourque! We have to add them under the conditional because they require the VPN to import and access that information. I believe we avoided this prior with Jenkins because it has access on the ST side of the firewall.